herokuscript.sh

```diff
@@ -0,0 +1,12 @@
+# Get latest docker image from Docker Hub
```

Comment:
It's worth adding a shebang and `set -e` at the top of script files to make sure they exit early if one of the commands fails.